src/helpers/ht_data.service.ts

```diff
@@ -70,11 +81,12 @@ export class HT_DataService {
             .map(key => (<HT_Data>HT_LocalStorage.getItem(key)).username);
     }
 
-    private setUsername(username: string): void {
+    private setUsername(username): void {
+        username = username.replace(/(^\")|("$)/gi, "");
```

Comment:
Me & Naveen investigated this and we think you are trying to remove the double quotes from the start and end of the string. It's confusing as we tried it on a regex testing website but can we just use ```replace('\"\', '')?

There's no need to use the global and case insensitive flag if you must use regex

Reply:
Its already being replaced in the calling code, so I removed this